db/migrate/20220805053443_populate_notification_data.rb

```diff
@@ -0,0 +1,20 @@
+class PopulateNotificationData < ActiveRecord::Migration[7.0]
+  include ActivityNotification
+  def change
+    Notification.find_each do |data|
```

Comment:
Do not use ActiveRecord models in migration

Reply:
Any alternative in which I can use `ActivityNotification` methods and adding values columns in `jsonb`.
I was getting stuck at it so tried this solution instead!